docs/HowTo/Configure/Configure-TLS.md

```diff
@@ -0,0 +1,54 @@
+description: Configure TLS
+<!--- END of page meta data -->
+
+# Configure TLS
+
+Hyperledger Besu supports TLS for inbound communication. For example, you can
+configure TLS for communication between EthSigner and Besu.
+
+Use the command line to configure TLS.
+
+Besu only supports password-protected PKSC #12 keystore files.
+
+**Prerequisites**:
+
+* Password-protected PKSC #12 keystore.
+* File containing the keystore password
+
+## Create Known Clients File
+
+Create a file (in this example, `knownClients`) that lists one or more clients
+with permission to connect to Besu. Use the format`<common_name> <hex-string>`
```

Comment:
This is also slightly misleading (and we might change its meaning in a future commit as well), basically we are enabling TLS "client authentication" on our endpoint. The "knownClients" file, at the moment, can be used to record self-signed client certificates fingerprint. At the moment, this file is **NOT** being used as permissions to connect to Besu (we will be bringing in a change in Besu code very soon which will make this file as a 'permission' file - but for now, this is not a permission file).